src/thunderbots/software/test/util/logger_test_node.cpp

```diff
@@ -0,0 +1,103 @@
+#include <gtest/gtest.h>
```

Comment:
Hmm, after looking at the repo we're not _super_ consistent with naming for test files, but could we rename this to `logger_test` or rename the class in this file to `LoggerTestNode` and rename the other RosTest files and classes accordingly? I'd be happy with either option

Reply:
Renamed to `logger_test`